requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests==2.18.1
+pep8==1.7.0
```

Comment:
`pep8` has been renamed to `pycodestyle` (but not aliased, le sigh, and no warning or anything…). @jeffcarp can you write a PR to change this to latest pycodestyle?

Reply:
![k](https://media1.giphy.com/media/Y01jP8QeLOox2/giphy.gif)